appscale/tools/admin_api/client.py

```diff
@@ -147,6 +147,45 @@ def delete_version(self, project_id, service_id, version_id):
 
     return operation_id
 
+  @retry(**RETRY_POLICY)
+  def patch_version(self, version, fields):
+    """ Patches (updates) a version.
+
+    Args:
+      version: A Version object.
+      fields: The set of fields to patch.
+    Returns:
+      A dictionary containing the deployment operation details.
+    Raises:
+      AdminError if the Admin API call was not successful.
+    """
+    versions_url = ('{prefix}/{project}/services/{service}/versions/{version}'
+                    .format(prefix=self.prefix, project=version.project_id,
+                            service=version.service_id, version=version.id))
+    headers = {
+      'AppScale-Secret': self.secret,
+      'Content-Type': 'application/json'
+    }
+    parameters = {
+      'updateMask': ','.join(fields)
+    }
+    query_parameters = '&'.join('='.join(item) for item in parameters.items())
```

Comment:
I would recommend using `urllib.urlencode(parameters)` for this use case. An alternative would be using the `params` argument to `requests.patch`.

Reply:
Updated to pass parameters directly to `requests.patch`. Thanks for the suggestion.